cmd/bundle/summarise.go

```diff
@@ -0,0 +1,67 @@
+package bundle
+
+import (
+	"encoding/json"
+	"errors"
+	"os"
+	"path/filepath"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/bundle/deploy/terraform"
+
+	"github.com/databricks/cli/bundle/phases"
+	"github.com/spf13/cobra"
+)
+
+func newSummariseCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "summarise",
```

Comment:
This is British spelling but we use American spellings throughout CLI, so it should be `summarize` instead

Reply:
Yeah we absolutely need to use American spelling here. Using `summary` would be a nice way to sidestep any discomfort with customers used to British spelling.